quickstart.md

```diff
@@ -698,7 +699,7 @@ Run skopeo to encrypt the image created in the previous section:
 
 ```
 sudo OCICRYPT_KEYPROVIDER_CONFIG=$(pwd)/attestation-agent/sample_keyprovider/src/enc_mods/offline_fs_kbs/ocicrypt.conf \
-skopeo copy docker-daemon:[REGISTRY_URL]:unencrypted \
+skopeo copy --insecure-policy docker-daemon:[REGISTRY_URL]:unencrypted \
```

Comment:
Might be good to add a note about what `--insecure-policy` is. Don't want to frighten anyone with the i-word.

Reply:
Added a note after the command.